repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasAbstractDefStoreV2.java

```diff
@@ -58,6 +59,22 @@
     private static final Pattern INTERNAL_NAME_PATTERN = Pattern.compile(INTERNAL_NAME_REGEX);
 
     public static final String ALLOW_RESERVED_KEYWORDS = "atlas.types.allowReservedKeywords";
+    public static final List<String> KEYWORDS_INVALID_FOR_TYPE_CREATION = new ArrayList() {
+        {
+            add("boolean");
+            add("byte");
+            add("short");
+            add("int");
```

Comment:
Instead of multiple add calls, make string[] array and then convert array to list, it will make it more readable. 
List al = Arrays.asList(geeks);